nixos/modules/services/web-apps/davis.nix

```diff
@@ -81,6 +81,8 @@ in
       type = lib.types.str;
     };
 
+    # NOTE: davis does not support php 8.4 yet
+    #       https://github.com/tchapi/davis/issues/195
```

Comment:
```suggestion
```
I think this comment is unnecessary. Let the package handle internal dependencies of davis.